bus/bus.go

```diff
@@ -98,6 +108,28 @@ type bus struct {
 	cs  ContractStore
 	css ContractSetStore
 	os  ObjectStore
+	ss  SettingStore
+}
+
+func (b *bus) loadDefaultSettings() (err error) {
+	rs := DefaultRedundancySettings()
+
+	// check min shards setting
+	if _, err = b.ss.Setting("MinShards"); isErrSettingsNotFound(err) {
+		err = b.ss.UpdateSetting("MinShards", fmt.Sprint(rs.MinShards))
```

Comment:
Not a fan of load methods that implicitly store data. We could just return the default values without writing them to the db.

Reply:
closing in favour of https://github.com/SiaFoundation/renterd/pull/55#discussion_r1043464948